Makefile

```diff
@@ -85,12 +85,13 @@ check-docs: generate-docs ## Check whether reference documentation needs to be g
 
 generate-docs: ## Generate command-line reference documentation
 	rm -rf ./docs/commands/*
-	go run ./cmd/rhoas docs --dir ./docs/commands --file-format adoc
+	go run ./cmd/rhoas docs --dir ./docs/commands --file-format md
 .PHONY: generate-docs
 
-generate-modular-docs: generate-docs ## Generate modular command-line reference documentation
-	SRC_DIR=$$(pwd)/docs/commands DEST_DIR=$$(pwd)/dist go run ./cmd/modular-docs
-.PHONY: generate-modular-docs
+generate-downstream-docs: ## Generate command-line reference documentation in adoc format
+	rm -rf ./docs/commands/*
+	go run ./cmd/rhoas docs --dir ./dist --file-format adoc
```

Comment:
Why is this removing `./docs/commands`, but then generating the documentation into a completely different directory?

Reply:
Very good question. I think I need to provide documentation for that in the contributing to make it clear.

So the new workflow will split upstream and downstream:

1) We will have markdown ./docs/commands/ (./docs/commands/rhoas-cluster.md). 
I did not push that markdown here due to number of files created that will obfuscate already large PR
Phase 2 for this would be to use that markdown with docusaurus website and create couple other placeholders like CLI guidelines etc.

2) We will have adoc/downstream pushed to different branch (only on releases) - for that we use ./dist/ folder that is gitignored so we can use it to push entire content to the new branch using github action.